docs/api/cases/cases-api-create.asciidoc

```diff
@@ -0,0 +1,234 @@
+[[cases-api-create]]
+== Create case API
+++++
+<titleabbrev>Create case</titleabbrev>
+++++
+
+Creates a case.
+
+=== Request
+
+`POST <kibana host>:<port>/s/<space_id>/api/cases`
+
+=== Prerequisite
+
+You must have `all` privileges for the *Cases* feature in the *Management*,
+*{observability}*, or *Security* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>.
+
+=== Path parameters
+
+`<space_id>`::
+(Optional, string) An identifier for the space. If it is not specified, the
+default space is used.
+
+=== Request body
+
+`connector`::
+(Required, object) An object that contains the connector configuration.
++
+.Properties of `connector`
```

Comment:
All fields of all connectors are required but the user can pass `null` if they want to not specify a value.

Reply:
I had put this phrase at the beginning of the "fields" description, but do you think it's likely to be overlooked?:

> If you want to omit any individual field, specify `null` as its value.

Reply:
Oh sorry, I missed that. I think it is fine 🙂.